### DISCUSSION TITLE
Is it possible to override max limit [256] jobs per workflow?

### DISCUSSION THREAD

**Discussion by wolsson7** [community/community#69683] [Actions] [answered]
### Select Topic Area

Question

### Body

I have a unique situation where several (around 300) jobs need to be run for a workflow. This worked up to 256 which I also read in the docs is the limit. But now the number of jobs have exceeded that. Is there any way to override this? A workaround or another way to have the same effect?

**Comment by santoshvandari** [comment, marked as answer]
As of my Knowledge, GitHub Actions has a hard limit of 256 jobs per workflow run. GitHub defined this limit and couldn't be overridden through configuration.

If you've exceeded this limit and need to run more than 256 jobs as part of your workflow, you can consider breaking your workflow into multiple workflows or using a different CI/CD tool.

The following are my suggestions to address your issue:

1. **Multiple Workflows:**  You can divide your tasks into multiple workflows, each with a subset of the jobs you need to run. As per your requirements, these workflows can be triggered sequentially or in parallel. While this doesn't increase the job limit per workflow, it allows you to manage a more significant number of jobs across multiple workflows.

2. **Alternative CI/CD Tools:**  If running a very high number of jobs concurrently is an ultimate requirement for your use case, you can explore other CI/CD tools that support a more significant number of parallel jobs. Tools like Jenkins, GitLab CI/CD, or Travis CI offer more flexibility.

Note: GitHub Actions may introduce changes in the future, so it's advisable to check the GitHub Actions documentation or GitHub's official announcements for any updates on job limits.

**Reply by wolsson7** [reply]
Ok, many thanks for a swift and good reply! :+1: I think the the multiple workflows will probably do the trick.
